test/testdrive/catalog.td

```diff
@@ -274,6 +274,8 @@ unacceptable schema name 'pg_bar'
 "mz_catalog, pg_catalog, public, mz_temp"
 ! SET search_path = foo
 parameter "search_path" cannot be changed
+! SET SCHEMA foo
+parameter "search_path" cannot be changed
```

Comment:
if one has a philosophical aversion to this error message changing the name, let me know, but fixing it would be pretty messy I think

Reply:
Nope, this is great. PostgreSQL lets `search_path` leak in its error messages for `SET SCHEMA` too.